var/spack/repos/builtin/packages/cloverleaf/package.py

```diff
@@ -36,6 +36,12 @@ class Cloverleaf(MakefilePackage):
     conflicts('build=serial', when='%aocc', msg="Currently AOCC supports only ref variant")
     conflicts('@1.1', when='%aocc', msg="AOCC support is provided from version v.1.3 and above")
 
+    @run_before('build')
+    def patch_for_reference_module(self):
+        if self.spec.satisfies("@master %aocc"):
+            fp = self.package_dir + "/aocc_support.patch"
```

Comment:
```suggestion
            fp = join_path(self.package_dir, "aocc_support.patch")
```

Reply:
Hi @tldahlgren, Thanks for the suggestions.